catalog/from-k8s/resource.go

```diff
@@ -372,6 +380,13 @@ func (t *ServiceResource) generateRegistrations(key string) {
 				}
 				seen[addr] = struct{}{}
 
+				for _, port := range subset.Ports {
```

Comment:
Will this still sync the NodePort value properly for NodePort services as in the now-deleted line 264?

Reply:
Good catch. Reverting to previous port calculations until we can look at this further.